app/controllers/application_controller.rb

```diff
@@ -32,6 +32,13 @@ class ApplicationController < ActionController::Base
     raise ActionController::RoutingError, 'Not Found'
   end
 
+  protected
+
+    def super_and_current_users
+      users = Role.find_by(name: 'superadmin')&.users.to_a
+      users << current_user if current_user && !users.include?(current_user)
```

Comment:
To resolve the nil I was getting from this method in creating a new tenant account it looks like it needs to return the `users` in case the guard clause evaluates to false:
```suggestion
      users << current_user if current_user && !users.include?(current_user)
      users
```